tutorials/continuous-aggs-tutorial.md

```diff
@@ -0,0 +1,239 @@
+
+# Tutorial: Continuous Aggregates
+
+One kind of query that occurs frequently when performing data analysis is
+aggregating data into summaries.
+
+For instance if we want to find out the average of a column for each day in the 
+dataset stored in a hypertable, we would run 
+
+``` sql
+SELECT time_bucket('1 day', timedimesion_column_name), avg(column_name), stddev(column_name)
+FROM hypertable
+GROUP BY time_bucket('1 day', timedimesion_column_name);
+```
+
+However, performing this query as written requires scanning all the data within
+the table, which can be inefficient if this query is called frequently.
+A continuous aggregate query computes the query results automatically at a user 
+specified time interval without the user having to repeatedly rerun the query. 
```

Comment:
`A continuous aggregate view pre-computes the query results automatically at a user 
specified time interval and materializes the results. When the user queries the view, the system
reads and processes the much-smaller materialized table. This speed up the query significantly. `

Reply:
like this wording. changed.